Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/Artifact/Factory/TestImpactTestEnumerationSuiteFactory.cpp

```diff
@@ -84,4 +87,74 @@ namespace TestImpact
             return testSuites;
         }
     } // namespace GTest
+
+    namespace Python
+    {
+        AZStd::vector<AZStd::pair<AZStd::string, AZStd::vector<TestEnumerationSuite>>> TestEnumerationSuitesFactory(
+            const AZStd::string& testEnumerationData)
+        {
+            const char newLineDelim = '\n';
+            size_t startLine;
+            size_t endLine = 0;
+            int count = 0;
+
+            AZStd::vector<AZStd::pair<AZStd::string, AZStd::vector<TestEnumerationSuite>>> pairs;
+            AZStd::map<AZStd::string, AZStd::vector<TestEnumerationSuite>> testSuiteMap;
```

Comment:
suggestion: If these types popup in more than one place it might be worth making an alias for them with the `using` keyword. That will make the code a lot more readable and remove some of the type noise

Reply:
Done!